js/__internal/grids/pivot_grid/data_controller/module.ts

```diff
@@ -7,7 +7,7 @@ import Class from '@js/core/class';
 import { format } from '@js/core/utils/string';
 import { deferUpdate } from '@js/core/utils/common';
 import { isDefined, isString } from '@js/core/utils/type';
-import { VirtualScrollController } from '@js/ui/grid_core/ui.grid_core.virtual_scrolling_core';
+import VirtualScrollControllerModule from '@js/__internal/grids/grid_core/virtual_scrolling/module_core';
```

Comment:
Let's change this import to:
```suggestion
import VirtualScrollControllerModule from '../../grid_core/virtual_scrolling/module_core';
```

Because we importing ts not js :)

Reply:
I think it's better to leave absolute path with `@js`, since it will be easier to find it via regex to change it to `@ts/grid_core/virtual_scrolling/module_core`